src/wsiToDcm.cpp

```diff
@@ -196,18 +198,33 @@ int WsiToDcm::dicomizeTiff(std::string inputFile, std::string outputFileMask,
     uint32_t batch = 0;
     std::vector<std::unique_ptr<Frame> > framesData;
     double downsampleOfLevel = downsample / multiplicator;
+    int64_t level_frameWidth;
+    int64_t level_frameHeight;
     dimensionDownsampling(
             frameWidth, frameHeight, levelWidth, levelHeight, retile, level,
             downsampleOfLevel, &frameWidthDownsampled, &frameHeightDownsampled,
-            &levelWidthDownsampled, &levelHeightDownsampled);
+            &levelWidthDownsampled, &levelHeightDownsampled, &level_frameWidth,
+            &level_frameHeight);
+
+  BOOST_LOG_TRIVIAL(debug) << "levelToGet: " << levelToGet;
+  BOOST_LOG_TRIVIAL(debug) << "multiplicator: " << multiplicator;
+  BOOST_LOG_TRIVIAL(debug) << "downsample: " << downsample;
+  BOOST_LOG_TRIVIAL(debug) << "downsampleOfLevel: " << downsampleOfLevel;
+
+  BOOST_LOG_TRIVIAL(debug) << "frameDownsampled: " << frameWidthDownsampled << ", "
+                             << frameHeightDownsampled;
+
+  BOOST_LOG_TRIVIAL(debug) << "levelDownsampled: " << levelWidthDownsampled << ", "
+                             << levelHeightDownsampled;
+
     while (y < levelHeight) {
       while (x < levelWidth) {
         assert(osr != nullptr && openslide_get_error(osr) == nullptr);
-        BOOST_LOG_TRIVIAL(debug)
-            << "x: " << x << " y: " << y << " level: " << level;
+        // BOOST_LOG_TRIVIAL(debug)
+        //    << "x: " << x << " y: " << y << " level: " << level;
```

Comment:
nit: Instead of commenting out this line it can be removed if it's not useful.

Reply:
Dan I found it..